### ISSUE TITLE
Allow the app to use local version of Prism

### ISSUE THREAD

**Issue by tbarn** [stoplightio/desktop#157]
Currently, the desktop app defaults to using the hosted version of Prism, which makes it hard to use local services

**Comment by lottamus** [comment]
![image](https://user-images.githubusercontent.com/7423098/37998624-dadd6366-31e4-11e8-954d-b7fc370d4031.png)

![image](https://user-images.githubusercontent.com/7423098/37998573-ad68de9c-31e4-11e8-881f-9c516a25dd10.png)


In the desktop App, you can configure what port Prism runs on. Would this solve the issue?

**Comment by tbarn** [comment]
would this work @pytlesk4?

**Comment by pytlesk4** [comment]
Nope, that would only be for the conductor. Instances still go through the hosted version, nothing we can do about that right now. Just need the bandwidth to get to it.
